misc/formatter/modules/formatter-core/src/test/resources/expressions/conditional/assert/conditional_expression_1.bal

```diff
@@ -5,4 +5,14 @@ function getValue(int v) returns int|() {
 public function foo() {
     int t = true ? 7 : 0;
     int u = getValue(6) ?: 8;
+
+    string var1 = true ?
+        "hello" : "hi";
+
+    string var2 = true ? "hello" :
+        "hi";
+
+    string var3 = true ?
+        "hello" :
+        "hi";
```

Comment:
Shall we add a test case for the scenario where only the end expression goes to a new line?

e.g.

```
string var3 = true ? "hello" :
        "hi";
```

and 
```
string var3 = true ? "hello" 
        : "hi";
```

Reply:
+1